### PR TITLE
FS-15 Stove

### PR DESCRIPTION
**Pull request by Jakllp**
- Added Stove (with knob - twist to turn on)
- Added cooking mechanic
- Burned will disappear
- No visual difference between cooked/not cooked yet

### CODE REVIEW

**Comment by Jakllp** [comment]
WAIT for merge until branch foodtruck-interior-assets is merged

**Review by Thyrexx — APPROVED** [review]

**Comment by Thyrexx** [comment]
Somehow it still says the files:
```
assets/models/top_bun.blend
assets/models/top_bun.blend1
``` 
have conflicts.
Although I already succesfully merged dev int this one and then this one into dev.
 I don't know what's up there but I will close this pull request manually now.

**Pull request closed without merging**
